templates/internet-of-things/smart-city.html

```diff
@@ -396,7 +396,7 @@ <h2>IoT OTA updates</h2>
           <div class="p-logo-section__item">
           {{ image (
             url="https://assets.ubuntu.com/v1/efca8490-intel-logo-0.png",
-            alt="",
+            alt="Intel logo",
```

Comment:
These four too please, then good to go
```suggestion
            alt="Intel",
```